service/src/main/java/bio/terra/landingzone/library/configuration/LandingZoneSamConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.landingzone.library.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "landingzone.sam")
+public class LandingZoneSamConfiguration {
```

Comment:
An unfortunate side effect of the "amalgam" approach: bean names must be unique. :( Hence `LandingZoneSamConfiguration`, `LandingZoneSamService`, etc

Reply:
Would `@Qualifier` help avoid this problem?

Reply:
We use `@Qualifier` for stairway which is indeed a shared component, but I'm not sure we want to share `SamService`, `JobService` between WSM and LZ because there are service-specific things in there. I think the right thing to do is probably split LZ into its own service as soon as reasonable..